src/distribution/cauchy.rs

```diff
@@ -0,0 +1,238 @@
+use distribution;
+use source::Source;
+
+/// A Cauchy distribution.
+///
+/// A Cauchy distribution (aka Lorentz or Cauchy–Lorentz distribution) is a continuous
+/// probability distribution with a location parameter `loc`, a scale parameter `gamma > 0`,
+/// and the following probability density function:
+///
+/// `p(x) = const / (1 + ((x - loc) / gamma)^2)`.
+///
+/// A Cauchy distribution is long tailed and has no well-defined mean or variance. It is
+/// unimodal with its mode at `loc`, around which it is symmetric. The ratio of two
+/// independent Gaussian distributed random variables is Cauchy distributed.
+///
+/// See [Wikipedia article on Cauchy
+/// distribution](https://en.wikipedia.org/wiki/Cauchy_distribution).
+#[derive(Clone, Copy, Debug)]
+pub struct Cauchy {
+    loc: f64,
+    gamma: f64,
+}
+
+impl Cauchy {
+    /// Create a Cauchy distribution with location `loc` and scale `gamma`.
+    ///
+    /// It should hold that `gamma > 0`.
+    #[inline]
+    pub fn new(loc: f64, gamma: f64) -> Self {
+        should!(gamma > 0.0);
+        Cauchy { loc, gamma }
+    }
+
+    // Return the location parameter
```

Comment:
Please see how it is given in other distributions. It should be three slashes and end with a period.